src/_sass/_variables.scss

```diff
@@ -42,10 +44,16 @@ $custom-select-indicator: url('data:image/svg+xml,<svg width="12" height="7" vie
 
 :root {
   --bs-primary: #{$primary};
+  --bs-primary-rgb: #{$primary-rgb};
   --bs-body-color: #{$primary-black};
   --bs-link-color: #{$primary};
+  --bs-link-color-rgb: #{$primary-rgb};
   --bs-body-font-family: "Poppins", sans-serif;
   --bs-link-hover-color: #{$dark-blue};
+  --bs-info-bg-subtle: #d5eef5;
+  --bs-info-rgb: 161, 216, 233; // border
+  --bs-warning-bg-subtle: #fff1d0;
+  --bs-warning-rgb: 254, 223, 149; // border
```

Comment:
These are used for the background and border colors of these boxes:
![image](https://github.com/user-attachments/assets/438b57a9-c25a-4400-aef2-6e2a27eab129)